include/ck_tile/ops/fmha/pipeline/block_fmha_pipeline_qr_ks_vs_async.hpp

```diff
@@ -114,6 +114,8 @@ struct BlockFmhaPipelineQRKSVSAsync
             {
                 return 1;
             }
+            else
```

Comment:
nit: safer with braces

Reply:
Changed !

Reply:
do we also want to ensure here that hdim is <= 512?